compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/declarations/impl/IrClassImpl.kt

```diff
@@ -99,4 +99,7 @@ class IrClassImpl(
         typeParameters.transform { it.transform(transformer, data) }
         declarations.transform { it.transform(transformer, data) }
     }
+
+    override fun toString(): String = descriptor.toString()
```

Comment:
Why you need toString override here and below? 
We have several utilities for debug purpose: ir2string() and dump()

Reply:
The debugger uses toString for formatting the little labels next to objects in the value explorer. It saves you 3-4 mouseclicks through the descriptors/names/owners etc in order to see e.g. the class name. 

I am completely open to suggestions!

Reply:
Actually you can specify custom rendered for any class in IJ debugger)

Reply:
In any case, please don't use descriptor's `toString()` here. 
- This will not work properly for declarations created or moved in lowerings.
- In general, we are moving away from descriptor use.

Reply:
Max's suggestion is precisely what I needed. Will drop both toString's from this PR.